src/Features/Core/Portable/InitializeParameter/AbstractInitializeMemberFromParameterCodeRefactoringProviderMemberCreation.cs

```diff
@@ -122,18 +124,35 @@ internal abstract partial class AbstractInitializeMemberFromParameterCodeRefacto
             }
         }
 
-        private IFieldSymbol CreateField(
-            IParameterSymbol parameter, ImmutableArray<NamingRule> rules, ImmutableArray<string> parameterNameParts)
+        private async Task<IFieldSymbol> CreateFieldAsync(
+            Document document,
+            CodeStyleOption<AccessibilityModifiersRequired> requireAccessibilityModifiers,
+            SyntaxNode functionDeclaration,
+            IParameterSymbol parameter,
+            ImmutableArray<NamingRule> rules,
+            ImmutableArray<string> parameterNameParts,
+            CancellationToken cancellationToken)
         {
             foreach (var rule in rules)
             {
                 if (rule.SymbolSpecification.AppliesTo(SymbolKind.Field, Accessibility.Private))
                 {
                     var uniqueName = GenerateUniqueName(parameter, parameterNameParts, rule);
 
+                    var accessibilityLevel = Accessibility.Private;
+                    if (requireAccessibilityModifiers.Value == AccessibilityModifiersRequired.Never || requireAccessibilityModifiers.Value == AccessibilityModifiersRequired.OmitIfDefault)
+                    {
+                        var model = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
note: i think you're getting the semantic model in order to determinet he containing type.  But you can get that straight from the "IParameterSymbol parameter" symbol.  Just call .ContainingType on that and pass it in.